core/admin/EE_Admin_Page.core.php

```diff
@@ -2503,11 +2503,9 @@ public function espresso_news_post_box()
                 // Get RSS Feed(s)
                 self::cached_rss_display(
                     'espresso_news_post_box_content',
-                    urlencode(
-                        apply_filters(
-                            'FHEE__EE_Admin_Page__espresso_news_post_box__feed_url',
-                            'https://eventespresso.com/feed/'
-                        )
+                    apply_filters(
+                        'FHEE__EE_Admin_Page__espresso_news_post_box__feed_url',
+                        'https://eventespresso.com/feed/'
```

Comment:
does it still work if you use `esc_url_raw()` ? 
considering it is a filtered url, might be good to have some kind of protection on it.

Reply:
Yep, switched it out to use it _raw